index.bs

```diff
@@ -10467,177 +10467,195 @@ asynchronous sections can't be reordered.
+				{{AudioNode}}s connected to this {{AudioNode}}. The resulting buffer is
+				called the <dfn>input buffer</dfn>.
+				<a href="#channel-up-mixing-and-down-mixing">Up or down-mix</a> it to
+				match if number of input channels of this {{AudioNode}}.
 
-			2. <a href="#computation-of-value">Compute the value(s)</a> of
-				this {{AudioParam}} for this block.
+			3. If this {{AudioNode}} is a <a>source node</a>,
+				<a href="computing-block"> compute a block of audio</a>, and
+				<a href="#available-for-reading">make it available for reading</a>.
 
-			3. <a>Queue a control message</a> to set the
-				{{[[current value]]}} slot of this {{AudioParam}}
-				to the last value computed in the preceding step.
+			4. Else, if this {{AudioNode}} is a <a>destination node</a>,
+				<a href="record-input">record the input</a> of this {{AudioNode}}.
 
-		2. If this {{AudioNode}} has any {{AudioNode}}s connected
-			to its input, <a href="#SummingJunction">sum</a> the buffers
-			<a href="#available">made available for reading</a> by all
-			{{AudioNode}}s connected to this {{AudioNode}}. The
-			resulting buffer is called the <dfn>input buffer</dfn>.
-			<a href="#channel-up-mixing-and-down-mixing">Up or down-mix</a>
-			it to match if number of input channels of this
-			{{AudioNode}}.
+		  5. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
+				{{AudioWorkletProcessor}}'s {{process}} method to
+				<a href="computing-block">compute a block of audio</a> with the argument
+				of <a href="#">input buffer</a>, output buffer and
+				<a href="#input-audioparam-buffer">input AudioParam buffer</a>.
 
-		3. If this {{AudioNode}} is a <a>source node</a>, <a href="computing">compute a block of audio</a>,
-			and <a href="#available">make it available for reading</a>.
+				1. Any {{Promise}} resolved within the execution of process method will
+					be queued into the microtask queue in the {{AudioWorkletGlobalScope}}.
 
-		4. Else, if this {{AudioNode}} is a <a>destination node</a>,
-			<a href="record">record the input</a> of this {{AudioNode}}.
+			6. Else, <a>process</a> the <a>input buffer</a>, and
+				<a href="#available-for-reading">make available for reading</a> the
+				resulting buffer.
 
-		5. Else, <a>process</a> the <a>input buffer</a>,
-			and <a href="#available">make available for reading</a> the resulting buffer.
+		6. <a>Atomically</a> increment {{BaseAudioContext/currentTime}} by 128 /
+			{{BaseAudioContext/sampleRate}}.
 
-	9. <a>Atomically</a> increment {{BaseAudioContext/currentTime}} by 128 / {{BaseAudioContext/sampleRate}}.
+		7. Return true.
 
-	10. Return true.
+	3. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#perform-a-microtask-checkpoint">
+		Perform a microtask checkpoint</a>.
 </div>
 
-<dfn id="mute">Muting</dfn> an {{AudioNode}} means that its
-output MUST be silence for the rendering of this audio block.
+<dfn id="mute">Muting</dfn> an {{AudioNode}} means that its output MUST be
```

Comment:
Did you just wrap these and following lines?  The preview diff shows no change in text here.  If you did, please undo this.